meetings/200325-board-meeting.md

```diff
@@ -104,3 +104,128 @@
+
+The board agreed on the fact that the chair and vice-chair should have some experience. However, this will be re-discussed in a separate meeting.
+
+
+## Dates for (remote) meetings in 2020
+
+The board think we should always allow remote meeting (even outside Covid-19 period).
+
+The board agreed to organize and lead 4 meetings per year. Next meeting will be decided at the end of a meeting (except for the first official board meeting where a doodle will be set-up.
+
+
+## Tasks
+
+### Instructor Training and online workshop
+
+- Is the board in charge of this?
+
+- Is it the role of the board to decide who can and who can't teach workshops?
+
+#### online workshop
+
+Carpentries is interested to know and wait for taking decision.
+
+But others, especially learners. But not too many instructors having time right now.
+
+To be flagged, the board needs to decide who gets the label (UiO@Carpentry) or should we accept any volunteers. This will be discussed in the next meeting.
+
+Everybody agreed on the fact that the board should be a facilitator.
+
+#### Communication
+
+Communication officer: communicate to community for online workshops. Be aware that there is possibilities to teach online.
+
+#### Funding
+
+Financial aspect: funding from data management project. No treasurer needed but a bit of money.
+
+#### On-boarding
+
+Several member of the board are new and there is a need for an on-boarding documentation (explain the usage of the mailing lists, etc.).
+
+Right now, we have many mailing lists and their usage is out of scope for this meeting. **Next meeting will be discussing about the email lists.**
+
+### Communication channels.
+
+- Microsoft team (--> difficult to use if not part of UiO)
+- github issues
+- gitter
+- mailing lists, etc.
+
+#### Communication channel usage
+
+- **Github**: documents that are public (nothing private or names from people that did not agree). Github issues can be used for governance chart, board minutes, etc. 
+- **UiO GSuite**: GSuite will be usef for non-public (yellow) data
+- **Email**: main communication channel for the board.
+
+
+### Urgent tasks
+
+- room situation, getting credit for teaching, Carpentry instructor, be more rewarding, etc... It will be discussed during the next board meeting.
+
+# AOB
+
+Mathworks (matlab) --> we all get this email. Should we offer matlab Carpentry workshop. Follow up with him.
+
+**Actions**: 
+- the new board will go through the governance documents and will update it accordingly.
+- The new board needs to check access to **uio-carpentry** github organization. Contact Annika if you needs to have write access to uio-carpentry github organization.
+- New board will organize appointment. Chair, vice-chair and communication officer;
+- Annika: doodle for setting up the next meeting.
+- Annika will write an on-boarding document for new board members.
+- Annika will follow up with Mathworks and see if they are willing to teach Carpentry matlab workshops at UiO.
```

Comment:
That was Elin's task, she has already done so, I will brief the Board the next time we meet about what they agreed on!